spec/raven/instance_spec.cr

```diff
@@ -215,7 +207,8 @@ describe Raven::Instance do
         instance.configuration.silence_ready = false
 
         instance.report_status
-        instance.logger.as(LoggerTest).infos.should contain(ready_message)
+        backend = instance.logger.as(LoggerTest).backend.as(Log::MemoryBackend)
+        backend.entries.map(&.message).should contain(ready_message)
```

Comment:
I'd prefer to use spec helpers introduced in https://github.com/crystal-lang/crystal/pull/9201

ditto all below

Reply:
That would be better indeed! Will do :)